proto/api/v1/auth_service.proto

```diff
@@ -28,6 +28,7 @@ message AuthStatus {
   storage.AuthProvider auth_provider = 5;
   storage.UserInfo user_info = 6;
   repeated UserAttribute user_attributes = 7;
+  string idp_token = 8;
```

Comment:
Why do we need this? I don't see you setting the IdP token in `/v1/auth/status` response so I wonder what exactly is an oversight here : )

Reply:
https://github.com/stackrox/stackrox/pull/9301/files#diff-3378a96291ab5d2f843a023eda609bb0fbffffda6adbf8e04297bf27f00b0335R312 it's not obvious, but `user` here is of type `AuthStatus`

Reply:
Indeed, thanks. Let's leave a comment above the line about the ephemeral nature of this field, i.e., that it is set only in few, specific contexts (you can omit that it's only set in test mode today) and one cannot rely on this field being always present in the response.